### PR TITLE
Issue #2664676: Add test cases for empty store.

### PR DESCRIPTION
**Pull request by miteshmap**
@mglaman: As per the task here are two test cases created for order and product. looking forward for your inputs. :)

### CODE REVIEW

File: modules/order/src/Tests/OrderEmptyStoreTest.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?php
+/**
```
Comment (bojanz) [inline modules/order/src/Tests/OrderEmptyStoreTest.php]: We no longer have @file docblocks in Commerce, Drupal has removed them from the coding standard.

File: modules/order/src/Tests/OrderEmptyStoreTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ * Contains \Drupal\commerce_order\Tests\OrderEmptyStoreTestBase,
+ * Contains \Drupal\commerce_order\Tests\OrderEmptyStoreTest.
+ */
+namespace Drupal\commerce_order\Tests;
+
+use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\Core\Url;
+use Drupal\field\Tests\EntityReference\EntityReferenceTestTrait;
+use Drupal\Core\Entity\EntityInterface;
+/**
```
Comment (bojanz) [inline modules/order/src/Tests/OrderEmptyStoreTest.php:8]: Missing empty line between the use statements and the docblock.

File: modules/order/src/Tests/OrderEmptyStoreTest.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ * Defines base class for Empty store test cases.
+ */
+abstract class OrderEmptyStoreTestBase extends CommerceTestBase {
```
Comment (bojanz) [inline modules/order/src/Tests/OrderEmptyStoreTest.php]: You should not be creating a new test base class, use OrderTestBase instead, add the rest in OrderEmptyStoreTest

File: modules/order/src/Tests/OrderEmptyStoreTest.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  }
+}
+
+
+/**
+ * Empty Store order page test.
+ *
+ * @group commerce
+ */
+class OrderEmptyStoreTest extends OrderEmptyStoreTestBase {
```
Comment (bojanz) [inline modules/order/src/Tests/OrderEmptyStoreTest.php]: A file can't contain more than one class.

File: modules/product/src/Tests/ProductEmptyStoreTest.php [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+namespace Drupal\commerce_product\Tests;
+
+use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\Core\Url;
+use Drupal\field\Tests\EntityReference\EntityReferenceTestTrait;
+use Drupal\Core\Entity\EntityInterface;
+/**
+ * Defines base class for Empty store test cases.
+ */
+abstract class ProductEmptyStoreTestBase extends CommerceTestBase {
```
Comment (bojanz) [inline modules/product/src/Tests/ProductEmptyStoreTest.php]: Same feedback as for orders.
